### DISCUSSION TITLE
Inclusion in official package sources of Debian project for installation via apt

### DISCUSSION THREAD

**Discussion by linux-lukas** [stuarthayhurst/alphabetical-grid-extension#66] [General]
While considering a distro change from Ubuntu to Debian (GNOME to GNOME), I noticed that their application probably cannot be installed via apt. Could one approach the Debian project here and ask whether your extension could be included in the official Debian package sources (stable)?

**Comment by stuarthayhurst** [comment]
I have no issues with this, but I don't really have time to do this myself. Open to anyone willing to maintain the Debian packaging.

**Reply by linux-lukas** [reply]
Okay, cool! I have comparatively small experience in programming, but could an automated process be implemented that when a new release comes out the Debian package sources say "Yo, new version, check it out" where the Debian project just has its internal processes?

**Reply by stuarthayhurst** [reply]
I'm not very familiar with Debian packing. I had a look in the past, but I couldn't figure it out. Considering the structure doesn't change much, once it's set up it would probably just require the source to be uploaded for each release, but I don't know enough to say for certain.
